commands/try-runtime/try-runtime.sh

```diff
@@ -13,6 +13,10 @@ main() {
   local chain="$1"
   local uri="$2"
```

Comment:
Wait, the args are defined as key-value map [in json file](https://github.com/paritytech/command-bot-scripts/blob/main/commands/try-runtime/try-runtime.cmd.json#L8-L11), but addressed as positional here.  
How is the order preserved?

Reply:
for now positioned, but later this will be turned into named as you were suggesting